truss/cli.py

```diff
@@ -228,6 +228,31 @@ def train(target_directory, build_dir, tag, var, vars_yaml_file):
     return tr.docker_train(build_dir=build_dir, tag=tag, variables=variables)
 
 
+@cli_group.command()
+@click.option(
+    "--var",
+    multiple=True,
+    help="""Training variables in key=value form where value is string.
+    For more complex values use vars_yaml_file""",
+)
+@click.option(
+    "--vars_yaml_file",
+    required=False,
+    help="Training variables from a yaml file",
+)
+@error_handling
+@echo_output
+def local_train(target_directory, build_dir, tag, var, vars_yaml_file):
```

Comment:
I think it's better to add a `local` arg to the train function above that defaults to `False` than to make another interface here. That way we can `truss train --local` just as we `truss predict --local`

Reply:
good point, I completely forgot about --local. Updating.